global.json

```diff
@@ -0,0 +1,7 @@
+{
+  "sdk": {
+    "version": "5.0.104",
```

Comment:
I'm wondering if we're being too specific here. I don't really like it when projects force me to download some specific version of the SDK. Is it that we require 5.0.1xx, or is it just that we don't currently want to pick up 6 on machines that have it?

As I understand it, `rollForward` defaults to `latestPatch`, which means that it won't accept a later feature band. I have SDKs 5.0.202 and 5.0.203 on my machine right now, which would be rejected by this `global.json`, forcing me to go and find an older SDK. Is that really necessary?

Could we set `rollForward` to `minor`? That way, if someone does have 5.0.104, or any later 5.0.1xx version, it'll use that, but if like me they don't have any 5.0.1xx versions but do have some 5.0.2xx versions, it'll pick the latest one of that, as I understand it.

Reply:
I just tested that `allowPrerelease: false` does not work together with other options. And If we optimistically assume that
1. People have at least SDK 5.x installed, and
2. Final version of SDK 6.x will eventually fix this problem

Then we can simplify it to:

```json
{
  "sdk": {
    "allowPrerelease": false
  }
}
```

Later, we can remove this file entirely once we confirm that the issue has been fixed in SDK 6.x